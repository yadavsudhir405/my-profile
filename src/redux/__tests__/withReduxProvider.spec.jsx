import React from 'react';
import { render } from '@testing-library/react';

import withRedux from '../withReduxProvider';

describe('WithReduxProvider', () => {
  it('should match snapshot', () => {
    const props = {
      router: {
        location: {
          pathname: '/login',
        },
      },
    };

    const component = props => <div {...props}>Redux</div>;
    const Redux = withRedux(component);
    const { container } = render(<Redux env="development" {...props} />);
    expect(container).toMatchSnapshot();
  });
});
