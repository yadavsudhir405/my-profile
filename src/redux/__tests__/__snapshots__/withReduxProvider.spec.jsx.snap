// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithReduxProvider should match snapshot 1`] = `
<div>
  <div
    env="development"
    router="[object Object]"
  >
    Redux
  </div>
</div>
`;
